src/axes/presets/scientific.typ

```diff
@@ -0,0 +1,153 @@
+#import "/src/cetz.typ": util, draw, styles, vector
+#import "../style.typ": default-style, _prepare-style, _get-axis-style
```

Comment:
There was some reason to use absolute paths, but idk. anymore…

Reply:
I'll leave it in for a separate refactor to deal with as I don't feel I know enough to properly do that